packages/utils/styling/src/getMarginAdjustment.win32.ts

```diff
@@ -1,39 +1,11 @@
-/**
- * Adjusts the margin of text so that it's centered within the layout area of the text.
- * Due to how Segoe UI lays out, the text does not appeared centered within the text
- * layout area without some fuding of margins.
- * @returns Adjustment necessary for text
- */
-export function getTextMarginTopAdjustment(): number {
-  return -1;
-}
-
-/**
- * Adjusts the margin of text so that it's centered within the layout area of the text.
- * Due to how Segoe UI lays out, the text does not appeared centered within the text
- * layout area without some fuding of margins.
- * @returns Adjustment necessary for text
- */
-export function getTextMarginBottomAdjustment(): number {
-  return 1;
-}
-
-/**
- * Adjusts the margin of text so that it's centered within the layout area of the text.
- * Due to how Segoe UI lays out, the text does not appeared centered within the text
- * layout area without some fuding of margins.
- * @returns Adjustment necessary for text
- */
-export function getTextMarginLeftAdjustment(): number {
-  return 1;
-}
+const margins = { marginTop: -1, marginBottom: 1, marginLeft: 1, marginRight: -1 };
```

Comment:
Another thought (not review blocking): 
I wonder if information like this could live in a specific theme, rather than globally through an extra package. I have this thought because there's no guarantee that win32 means Segoe UI: you only really know you have Segoe if you have a theme that uses Segoe. With the apple theme, I had [component specific overrides](https://github.com/microsoft/fluentui-react-native/blob/78a02005cdd60b263b1c3f9ecec5187e9a4a9931/packages/theming/apple-theme/src/appleTheme.macos.ts#L15). With global/control/alias tokens, I feel like the role of themes is changing so I don't quite understand if doing stuff like component specific overrides makes sense anymore.

Reply:
Hmm yeah, that makes sense. I'll open an issue to track this.